game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProTerritoryValueUtils.java

```diff
@@ -49,13 +49,6 @@ public static double findTerritoryAttackValue(final PlayerId player, final Terri
     return value;
   }
 
-  public static Map<Territory, Double> findTerritoryValues(final PlayerId player,
```

Comment:
This is one of the main keys of this PR was to remove this method and refactor areas that used it as its very slow on large maps.